plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/plugin.xml

```diff
@@ -83,7 +83,7 @@
                id="com.redhat.fabric8analytics.lsp.eclipse.ui.Authorize">
             <command
                   commandId="com.redhat.fabric8analytics.lsp.eclipse.ui.authorizetoken"
-                  icon="icons/osio-16x16.png"
+                  icon="icons/openshift_logo.png"
```

Comment:
check how openshift_logo is showing up as you need 16*16 looks like

Reply:
can't check on my system, right now. It crashed it. Maybe once I get the new system. :)